__tests__/app.test.js

```diff
@@ -165,3 +166,85 @@ describe("PATCH article", () => {
       });
   });
 });
+
+describe("GET articles", () => {
+  test("200: get all articles including property comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length > 0).toBe(true);
```

Comment:
So because this data gets reseeded before each test and you can see your test data you can safely be more specific about your length check - I would definitely be checking that the array length is exactly X number so that you aren't getting a false positive where only one article is coming back and it should've been 12 or something.